server/controllers/admin/enterprises.js

```diff
@@ -0,0 +1,76 @@
+/**
+* Enterprises Controller
+*
+* This controller is responsible for creating and updating Enterprises.  Every
+*/
+var db = require('../../lib/db');
+
+
+exports.list = function list(req, res, next) {
+  'use strict';
+
+  var sql;
+
+  sql =
+    'SELECT id, name, abbr, email, po_box, phone, location_id, logo, currency_id FROM enterprise';
+
+  db.exec(sql)
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+};
+
+// get Enterprises
+function helperGetEnterprise(id, codes) {
+  'use strict';
+
+  var sql;
+
+  sql =
+    'SELECT id, name, abbr, email, po_box, phone, location_id, logo, currency_id FROM enterprise ' +
+    'WHERE id = ?;';
+
+  return db.exec(sql, [id])
+  .then(function (rows) {
+
+    if (!rows.length) {
+      throw codes.NOT_FOUND;
+    }
+  });
+}
+
+// POST /enterprises
+exports.create = function create(req, res, next) {
+  'use strict';
+
+  var enterprise = req.body.enterprises,
```

Comment:
The variable attached to `req.body` should be called `enterprise`, not `enterprises`.